src/sentry/services/hybrid_cloud/app/__init__.py

```diff
@@ -143,9 +154,10 @@ def serialize_sentry_app(self, app: SentryApp) -> RpcSentryApp:
             slug=app.slug,
             uuid=app.uuid,
             events=app.events,
-            webhook_url=app.webhook_url,
+            webhook_url=app.webhook_url or "",
```

Comment:
General question: is this how we want to handle mismatches in nullability between ORM and RPC models?

That is, the ORM `SentryApp` class has

```python
    webhook_url = models.URLField(max_length=512, null=True)
```

but `RpcSentryApp` maps that onto a non-nullable `webhook_url: str = ""`.

Assuming we prefer to represent it that way in the RPC interface, the it makes sense to convert it to the default value during serialization. Unless we want to make the RPC model more faithful to the ORM schema, and change it to `webhook_url: Optional[str] = None`?

Reply:
That's likely fair.

Reply:
One partial reason I had originally written this way was, the orm types don't really enforce Nullability in mypy.  When you swap this out, you may?  Have to type fight a little bit.